OpenSim/Common/Exception.h

```diff
@@ -266,6 +266,11 @@ class IOError : public Exception {
     using Exception::Exception;
 };
 
+class NotImplementedYet : public Exception {
+public:
```

Comment:
Thanks @aseth1 There are multiple use cases, some could be for a branch in the code inside a method and others where a full method isn't implemented or overriden. The first case is handled by the base class as is. For the sake of keeping this PR as narrow as possible. I undid this change and will have a separate PR for it since I ended up not using it anyway.